cmd/melt/main.go

```diff
@@ -238,3 +255,30 @@ func completeColor(truecolor, ansi256, ansi string) string {
 	}
 	return ansi
 }
+
+// setLanguage sets the language of the big39 mnemonic seed.
+func setLanguage(language string) error {
+	switch strings.ToLower(language) {
+	case "chinese-simplified":
+		bip39.SetWordList(wordlists.ChineseSimplified)
+	case "chinese-traditional":
```

Comment:
There’s no good ISO code for traditional Chinese, but if you want to be complete, you could also match against `zh_HANT`. If we decide to do that I'd also match simplified against `zh_HANS`.

Reply:
Sure, why not